data/Aeneid/Aeneid03.textproto

```diff
@@ -8924,8 +8924,110 @@ verse {
   number: 87
   text: "Pergama, relliquiās Danaum atque immītis Achillī."
   norm: "pergama relliquiās danaum atque immītis achillī"
-  raw_pron: "pergama rellikwiaːs danawm atkwe immiːtis akilliː"
-  defective: true
+  raw_pron: "pergama rellikwiaːs danaũː atkwe immiːtis akilliː"
```

Comment:
I'm glad this is scanning now, but do you know what was changed?

Reply:
I'm pretty certain it was because I ran the grammar from #98 (which accounts for the pronounciation change from danawm to danaũː) onto this file.

Reply:
Oh right, this is the line we were discussing earlier.